src/main/java/space/obminyashka/items_exchange/controller/request/AdvertisementFindThumbnailsRequest.java

```diff
@@ -0,0 +1,30 @@
+package space.obminyashka.items_exchange.controller.request;
+
+import io.swagger.v3.oas.annotations.Parameter;
+import lombok.*;
+
+import javax.validation.constraints.PositiveOrZero;
+import java.util.UUID;
+
+import static space.obminyashka.items_exchange.util.ResponseMessagesHandler.ValidationMessage.*;
+
+@Getter
+@Setter
+public class AdvertisementFindThumbnailsRequest {
```

Comment:
You can omit **Thumbnails** from the name because that would be the return parameter of the requested endpoint